packages/itwinui-react/src/core/Table/Table.tsx

```diff
@@ -910,6 +893,7 @@ export const Table = <
             ...style,
           },
         })}
+        onScroll={() => updateStickyState()}
```

Comment:
I'm wondering if we still need this much JS for sticky columns. It might be possible to simplify

(Doesn't need to be handled in this PR but worth investigating later)

Reply:
Sounds good, added it to the after-PR TODOs.